### DISCUSSION TITLE
Q&Aトライアル

### DISCUSSION THREAD

**Discussion by takeoverjp** [seigot/burger_war_dev#5] [Q&A] [answered]
カテゴリが選べる。何が変わるんだろう。

**Comment by takeoverjp** [comment, marked as answer]
回答してみる

**Reply by seigot** [reply]
issueとどう使い分けるのがいいんだろう

**Reply by takeoverjp** [reply]
Q&Aの場合、Answerにマークできるから、明確な回答が得られそうなものはQ&Aってことですかねぇ。
（対応している環境はなに？とか）
で、バグっぽかったらIssue?
（環境構築手順通りやったらこんなエラーが出ましたとか）
まぁ後からでも変えれそうなので、適当で汗

**Reply by takeoverjp** [reply]
あ、カテゴリは変えれるけどissueには変えれなさそうですね（参照はできる）
失礼しました。。
